tensorflow/lite/toco/tflite/operator.cc

```diff
@@ -1060,6 +1076,8 @@ class Lstm : public BuiltinOperator<LstmCellOperator, ::tflite::LSTMOptions,
       case LstmCellOperator::KERNEL_BASIC:
         // KERNEL_BASIC was added in version 2.
         return 2;
+      default:
+        return -1;
```

Comment:
this was intentionally left out to have a compiler error if we miss an case

Reply:
Got it, I will leave some comments here :)